Add important note about known issue

andreiztm opened the pull request.

e0i commented:
The PR could also use some copy editing. Thank you for your contribution, @andreiztm 

@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

andreiztm commented:
> This is mentioned under https://docs.microsoft.com/en-us/windows/deployment/windows-autopilot/known-issues please link to there not directly to the blog. There are also some typos here so please re-read your note carefully. Also instead of the word "execute" we use the word "run" due to the geopolitical ramifications of 'execute' which I know might sounds silly but it can be translated in odd ways if used.

Thanks for the feedback. Not silly at all, modified that into "will run". I cannot link directly to an entry on the known issues page (thanks for sharing didn't know about this) and the particular entry for this issue also links to Michael's blog, so it just seems like an extra step to get to the source material. 

Noticed "syeprep" typo, but this was from before my PR, corrected it in the above important note as well.

JohanFreelancer9 commented on windows/deployment/windows-autopilot/existing-devices.md:
```diff
@@ -205,7 +205,10 @@ See the following examples.
 
      >[!IMPORTANT]
-     >The Autopilot for existing devices task sequence will run the **Prepare Windows for capture** action which calls the System Preparation Tool (syeprep). This action will fail if the target machine is joined to a domain.
+     >The Autopilot for existing devices task sequence will run the **Prepare Windows for capture** action which calls the System Preparation Tool (sysprep). This action will fail if the target machine is joined to a domain.
```
```suggestion
     >The Autopilot for existing devices task sequence will run the **Prepare Windows for capture** action which uses the System Preparation Tool (sysprep). This action will fail if the target machine is joined to a domain.
```

illfated replied:
Please add the following change by clicking the **Commit suggestion** button :
```suggestion
     > The Autopilot for existing devices task sequence will run the **Prepare Windows for capture** action which uses the System Preparation Tool (sysprep). This action will fail if the target machine is joined to a domain.
```

JohanFreelancer9 commented on windows/deployment/windows-autopilot/existing-devices.md:
```diff
@@ -205,7 +205,10 @@ See the following examples.
+     >The Autopilot for existing devices task sequence will run the **Prepare Windows for capture** action which calls the System Preparation Tool (sysprep). This action will fail if the target machine is joined to a domain.
+     
+     >[!IMPORTANT]
+     >The System Preparation Tool (sysprep) will run with the /Generalize parameter which on Windows 10 1903 & Windows 10 1909 versions will delete the AutoPilot profile file and the machine will boot into OOBE phase instead of AutoPilot. Please see Michael Niehaus's blog with instructions [A challenge with Windows Autopilot for existing devices and Windows 10 1903](https://oofhours.com/2019/09/1 ...
```
```suggestion
     >The System Preparation Tool (sysprep) will run with the /Generalize parameter which, on Windows 10 versions 1903 and 1909, will delete the Autopilot profile file and the machine will boot into OOBE phase instead of Autopilot phase. To fix this issue, please see [Windows Autopilot - known issues](https://docs.microsoft.com/windows/deployment/windows-autopilot/known-issues)
```

illfated replied:
Please add the following change by clicking the **Commit suggestion** button :
```suggestion
     > The System Preparation Tool (sysprep) will run with the /Generalize parameter which, on Windows 10 versions 1903 and 1909, will delete the Autopilot profile file and the machine will boot into OOBE phase instead of Autopilot phase. To fix this issue, please see [Windows Autopilot - known issues](https://docs.microsoft.com/windows/deployment/windows-autopilot/known-issues).
```

e0i commented:
@andreiztm 

Please let us know whenever you follow up with copy-editing suggestions. Thanks.

andreiztm commented:
> @andreiztm
> 
> Please let us know whenever you follow up with copy-editing suggestions. Thanks.

I don;t know how to check this or answer you. I did the changes mentioned in this commit: https://github.com/MicrosoftDocs/windows-itpro-docs/pull/6032/commits/1e394cff0eda86338ec81783ef8c297f327b4ae2

Is it ok?

illfated commented on windows/deployment/windows-autopilot/existing-devices.md:
```diff
@@ -205,7 +205,10 @@ See the following examples.
    - Click **Next**, and then on the Configure Network page choose **Join a workgroup** and specify a name (ex: workgroup) next to **Workgroup**.
 
      >[!IMPORTANT]
```
Please add the following change by clicking the **Commit suggestion** button :
```suggestion
     > [!IMPORTANT]
```

andreiztm commented:
@illfated how is it looking now? Thanks.

illfated commented:
Looks quite good, I hope you will forgive me for asking you to add 1 more spacing.

illfated commented on windows/deployment/windows-autopilot/existing-devices.md:
```diff
@@ -204,8 +204,11 @@ See the following examples.
+     > [!IMPORTANT]
+     > The Autopilot for existing devices task sequence will run the **Prepare Windows for capture** action which uses the System Preparation Tool (sysprep). This action will fail if the target machine is joined to a domain.
+     
      >[!IMPORTANT]
```
MarkDown indent marker spacing wanted:
```suggestion
     > [!IMPORTANT]
```

andreiztm commented:
Ok

andreiztm commented:
Thank you.

e0i commented:
@andreiztm Thank you.

@greg-lindsay Can you approve the recent changes? Thank you.

andreiztm commented:
Could we please review and approve this?

illfated commented:
@andreiztm : Sure, it will be done as soon as the PM has got time to finalize this.

In the meanwhile, may we ask @greg-lindsay to complete his review and approve the current content, just to signal that there is no outstanding changes required?

@JohanFreelancer9 : May I ask you to add your approval mark, if you find the current content agreeable?

JohanFreelancer9 commented:
Looks good.

illfated commented:
#sign-off
